service/controller/v2/resource/chartstatus/create.go

```diff
@@ -0,0 +1,48 @@
+package chartstatus
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/chart-operator/service/controller/v2/key"
+)
+
```

Comment:
The new resource.

Reply:
So the whole purpose of it is to update the status of the chart config yes? Why don't we manage the status within the resource that installs the chart based on the chart config? Is that a conscious design decision to have these resource implementations separated? Can you say more about this? The status magic is pretty new and I try to get an idea on how it should be.

Reply:
It feels cleaner to me to have a separate resource. So chart handles installing the chart and chartstatus handles setting the status.

At the moment the chart resource is still quite simple. But as we extend it with more features that is likely to change.

That's my take on it but happy to get more opinions.

Reply:
Makes sense. Is it that we then execute the resource and then its status pendant? 

```
chartResource,
chartStatusResource,
```

I would then go with the same in the `node-operator` because we want to have the node status managed there as well. 

```
nodeResource,
nodeStatusResource,
```

Reply:
Yes the status resource runs after the main resource. IMO this is a good approach for this.